Current snapshots won't build on many bsd systems without the following
patch, as machine/endian.h cannot be included without including sys/types.h
first.


2000-12-27	Marc Espie <espie@openbsd.org>
	* configure.in:  Check for sys/types.h.
	* configure:  Rebuild.
	* libmath/mathconf.h:  Include sys/types.h prior to
	machine/endian.h if available.

--- libstdc++-v3/configure.in.orig	Thu Jun 14 10:47:35 2001
+++ libstdc++-v3/configure.in	Mon Jul  9 08:43:44 2001
@@ -204,7 +204,7 @@ else
 
   # Check for available headers.
   AC_CHECK_HEADERS([nan.h ieeefp.h endian.h sys/isa_defs.h machine/endian.h \
-  machine/param.h sys/machine.h fp.h locale.h float.h inttypes.h gconv.h])
+  machine/param.h sys/machine.h sys/types.h fp.h locale.h float.h inttypes.h gconv.h])
 
   GLIBCPP_CHECK_COMPILER_FEATURES
   GLIBCPP_CHECK_LINKER_FEATURES
